settings.yml

```
@@ -0,0 +1,25 @@
+---
+lib_name: lib_xua
+project: '{{lib_name}}'
+title: '{{lib_name}}: USB Audio Components Library'
```

```suggestion
title: '{{lib_name}}: USB Audio components library'
```

Just follow rst style guide doc